packages/language-server/src/plugins/typescript/language-service.ts

```diff
@@ -60,22 +60,31 @@ export async function forAllLanguageServices(cb: (service: LanguageServiceContai
  */
 export async function getLanguageServiceForTsconfig(
 	tsconfigPath: string,
-	docContext: LanguageServiceDocumentContext
+	docContext: LanguageServiceDocumentContext,
+	workspaceUris: string[]
 ): Promise<LanguageServiceContainer> {
 	let service: LanguageServiceContainer;
 	if (services.has(tsconfigPath)) {
 		service = await services.get(tsconfigPath)!;
 	} else {
-		const newService = createLanguageService(tsconfigPath, docContext);
+		const newService = createLanguageService(tsconfigPath, docContext, workspaceUris);
 		services.set(tsconfigPath, newService);
 		service = await newService;
 	}
 
 	return service;
 }
 
-async function createLanguageService(tsconfigPath: string, docContext: LanguageServiceDocumentContext) {
-	const workspaceRoot = tsconfigPath ? dirname(tsconfigPath) : '';
+async function createLanguageService(
+	tsconfigPath: string,
+	docContext: LanguageServiceDocumentContext,
+	workspaceUris: string[]
+) {
+	const tsconfigRoot = tsconfigPath ? dirname(tsconfigPath) : process.cwd();
```

Comment:
Changed this back to `process.cwd()` because otherwise, getting types from Astro could fail in certain cases